bindata/network/multus/multus.yaml

```diff
@@ -133,6 +133,24 @@ spec:
           value: "/usr/src/multus-cni/rhel8/bin/"
         - name: DEFAULT_SOURCE_DIRECTORY
           value: "/usr/src/multus-cni/bin/"
+      - name: egress-router-binary-copy
+        image: "quay.io/dmellado/egress-router-test:latest"
```

Comment:
Make sure this winds up using the templated `{{.ImageNameHere}}` when you're ready to take this out of WIP